### ISSUE TITLE
double --caption_dropout_every_n_epochs="3"

### ISSUE THREAD

**Issue by g558800** [bmaltais/kohya_ss#554]
Why do I print training command, it looks like this:
accelerate launch --num_cpu_threads_per_process=2 "train_network.py" --enable_bucket --pretrained_model_name_or_path="/root/autodl_tmp/chilloutmix_NiPrunedFp32Fix.safetensors" --train_data_dir="/root/autodl-tmp/train_output/img" --resolution=768,768 --output_dir="/root/autodl-tmp/train_output/model" --logging_dir="/root/autodl-tmp/train_output/log" --network_alpha="80" --save_model_as=safetensors --network_module=lycoris.kohya --network_args "conv_dim=1" "conv_alpha=1" "algo=lora" --text_encoder_lr=0.00005 --unet_lr=0.00001 --network_dim=160 --gradient_accumulation_steps=4 --output_name="wengmeiling_girl" --lr_scheduler_num_cycles="18" --learning_rate="0.000001" --lr_scheduler="constant_with_warmup" --lr_warmup_steps="1228" --train_batch_size="4" --max_train_steps="12285" --save_every_n_epochs="1" --mixed_precision="fp16" --save_precision="fp16" --cache_latents --optimizer_type="AdamW" --max_data_loader_n_workers="6" --caption_dropout_every_n_epochs="3" --caption_dropout_every_n_epochs="3" --bucket_reso_steps=64 --shuffle_caption --xformers --bucket_no_upscale --noise_offset=0.1

double "--caption_dropout_every_n_epochs="3"  ".There is no parameter for 'Rate of caption dropout'.

**Comment by bmaltais** [comment]
This look like a bug... I will look into it once electricity is back. Currently going through the result of an ice storm around my place.

**Comment by g558800** [comment]
ok,thank you!

**Comment by bmaltais** [comment]
OK... this was caused by an entry that was supposed to handle the caption_dropout_rate. This will actually enable proper caption_dropout_rate during training... before it was just not used... doh.

**Comment by bmaltais** [comment]
Fixed via: https://github.com/bmaltais/kohya_ss/commit/1f153629fe65b5a634d07a0a99e022e3b54d9cdf
